addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4761,7 +4762,33 @@ registry.sizing = SnippetOptionWidget.extend({
                 resizeResolve();
                 self.trigger_up("enable_loading_effect");
 
+                // Check whether there has been a resizing.
                 if (directions.every(dir => dir.begin === dir.current)) {
+                    const mouseupTime = new Date();
+                    // Mouse move duration in milliseconds.
+                    const mouseHeldDuration = mouseupTime - mousedownTime;
+                    // If there has been no resizing and the duration of the
+                    // mouse held has been less than 500 milliseconds. We assume
+                    // that the user wanted to click on the snippet behind the
+                    // handle (e.g. select an "Over The Content" header when it
+                    // is entirely covered by the handle of the first snippet
+                    // of the page).
```

Comment:
- `If there has been no resizing and the duration of the mouse held has been less than 500 milliseconds. We assume` 
  => To make it shorter and clearer, I would reformulate it like this: `If no resizing happened and if the mouse was pressed less than 500 ms, we assume...`

- `on the snippet behind the handle` => `on the element behind the handle` (again because it is not necessarily a snippet, it could be a carousel indicator, which is not a snippet) :wink:  

- `(e.g. select an "Over The Content" header when it is entirely covered by the handle of the first snippet of the page).` => You should remove this, because you should not mention the header here, since we are in web_editor and the header is specific to website :slightly_smiling_face: